### DISCUSSION TITLE
Traveler's Tundra Mammoth

### DISCUSSION THREAD

**Discussion by fendwe** [chromiecraft/chromiecraft#4063] [Suggestions]
Would it be possible to make the [Traveler's Tundra Mammoth](https://wowgaming.altervista.org/aowow/?item=44235) available to the adventure enthusiast community?

**Comment by gettheboom** [comment]
No. Would be wrong for so many reasons to launch it now...

**Comment by Cooties-cc** [comment]
Probably not gonna happen. Its based on an achievement, and the scaling back to 60 from 80 in costs of 16-20k gold would be staggeringly high. It would also be very frowned upon if put in the donate shop. I assume you are wanting it for the portable vendor/repair, but for this you'll have to grab engineering or know one to drop a repair bot. There is also a tbc ver repair bot, but for more "easy" portable ones, we won't see till wrath content.
